app/code/Magento/Directory/Model/Currency.php

```diff
@@ -378,7 +390,8 @@ public function getConfigDefaultCurrencies()
      */
     public function getConfigBaseCurrencies()
     {
-        $defaultCurrencies = $this->_getResource()->getConfigCurrencies($this, self::XML_PATH_CURRENCY_BASE);
+        $defaultCurrencies = $this->_getResource()->getConfigCurrencies($this, self::XML_PATH_CURRENCY_BASE) ?:
```

Comment:
What happens in the case that you have multiple stores, 1 with a database value and one without. In this case I guess we need to merge the values here.

Reply:
Fixed.